[2024-07-12T01:47:49.045+0530] {processor.py:161} INFO - Started process (PID=36630) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:47:49.046+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py for tasks to queue
[2024-07-12T01:47:49.048+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:47:49.048+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:47:49.069+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:47:49.068+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py", line 13, in <module>
    from scripts.raw_load_investments import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'scripts'
[2024-07-12T01:47:49.069+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:47:49.089+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py took 0.053 seconds
[2024-07-12T01:48:19.317+0530] {processor.py:161} INFO - Started process (PID=37114) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:48:19.318+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py for tasks to queue
[2024-07-12T01:48:19.320+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:19.320+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:48:19.339+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:19.338+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py", line 13, in <module>
    from scripts.raw_load_investments import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'scripts'
[2024-07-12T01:48:19.339+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:48:19.359+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py took 0.049 seconds
[2024-07-12T01:48:50.812+0530] {processor.py:161} INFO - Started process (PID=37497) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:48:50.813+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py for tasks to queue
[2024-07-12T01:48:50.815+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:50.814+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:48:50.842+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:50.841+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py", line 13, in <module>
    from scripts.raw_load_investments import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'scripts'
[2024-07-12T01:48:50.843+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:48:50.869+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py took 0.065 seconds
[2024-07-12T01:48:55.647+0530] {processor.py:161} INFO - Started process (PID=37560) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:48:55.648+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py for tasks to queue
[2024-07-12T01:48:55.652+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:55.651+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:48:55.676+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:55.675+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py", line 13, in <module>
    from models.raw_models.raw_load_investments import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'models'
[2024-07-12T01:48:55.677+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:48:55.701+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py took 0.062 seconds
[2024-07-12T01:49:26.107+0530] {processor.py:161} INFO - Started process (PID=38003) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:49:26.108+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py for tasks to queue
[2024-07-12T01:49:26.109+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:49:26.109+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:49:26.126+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:49:26.125+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py", line 13, in <module>
    from models.raw_models.raw_load_investments import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'models'
[2024-07-12T01:49:26.127+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:49:26.144+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py took 0.044 seconds
[2024-07-12T01:49:57.001+0530] {processor.py:161} INFO - Started process (PID=38331) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:49:57.002+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py for tasks to queue
[2024-07-12T01:49:57.004+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:49:57.004+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:49:57.020+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:49:57.019+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py", line 13, in <module>
    from models.raw_models.raw_load_investments import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'models'
[2024-07-12T01:49:57.021+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:49:57.037+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py took 0.043 seconds
[2024-07-12T01:50:27.713+0530] {processor.py:161} INFO - Started process (PID=38740) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:50:27.714+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py for tasks to queue
[2024-07-12T01:50:27.716+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:50:27.716+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:50:27.735+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:50:27.733+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py", line 13, in <module>
    from models.raw_models.raw_load_investments import create_tables,tuncate_table,insert_table
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/models/raw_models/raw_load_investments.py", line 6, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
[2024-07-12T01:50:27.735+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:50:27.753+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py took 0.047 seconds
[2024-07-12T01:50:57.925+0530] {processor.py:161} INFO - Started process (PID=39180) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:50:57.926+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py for tasks to queue
[2024-07-12T01:50:57.928+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:50:57.927+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:50:57.948+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:50:57.946+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py", line 13, in <module>
    from models.raw_models.raw_load_investments import create_tables,tuncate_table,insert_table
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/models/raw_models/raw_load_investments.py", line 6, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
[2024-07-12T01:50:57.948+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:50:57.967+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py took 0.049 seconds
[2024-07-12T01:51:28.037+0530] {processor.py:161} INFO - Started process (PID=39578) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:51:28.038+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py for tasks to queue
[2024-07-12T01:51:28.040+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:51:28.039+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:51:28.058+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:51:28.057+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py", line 13, in <module>
    from models.raw_models.raw_load_investments import create_tables,tuncate_table,insert_table
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/models/raw_models/raw_load_investments.py", line 6, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
[2024-07-12T01:51:28.059+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:51:28.077+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py took 0.046 seconds
[2024-07-12T01:51:59.340+0530] {processor.py:161} INFO - Started process (PID=40175) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:51:59.342+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py for tasks to queue
[2024-07-12T01:51:59.345+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:51:59.344+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:52:00.138+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:00.137+0530] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-07-12T01:52:00.153+0530] {processor.py:840} INFO - DAG(s) 'raw_load_investments' retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:52:00.390+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:00.390+0530] {override.py:1847} INFO - Created Permission View: can delete on DAG:raw_load_investments
[2024-07-12T01:52:00.413+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:00.412+0530] {override.py:1847} INFO - Created Permission View: can read on DAG:raw_load_investments
[2024-07-12T01:52:00.435+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:00.434+0530] {override.py:1847} INFO - Created Permission View: can edit on DAG:raw_load_investments
[2024-07-12T01:52:00.436+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:00.435+0530] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-12T01:52:00.672+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:00.671+0530] {dag.py:3118} INFO - Creating ORM DAG for raw_load_investments
[2024-07-12T01:52:00.673+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:00.673+0530] {dag.py:3954} INFO - Setting next_dagrun for raw_load_investments to None, run_after=None
[2024-07-12T01:52:00.724+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py took 1.403 seconds
[2024-07-12T01:52:30.851+0530] {processor.py:161} INFO - Started process (PID=40872) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:52:30.852+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py for tasks to queue
[2024-07-12T01:52:30.854+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:30.854+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
[2024-07-12T01:52:31.347+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:31.347+0530] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-07-12T01:52:31.356+0530] {processor.py:840} INFO - DAG(s) 'raw_load_investments' retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_investments.py
